platform/commonUI/browse/test/InspectorPaneControllerSpec.js

```diff
@@ -71,7 +73,14 @@ define(
                 mockDomainObject.hasCapability.andReturn(true);
                 mockDomainObject.getCapability.andReturn(mockStatusCapability);
 
-                controller = new InspectorPaneController(mockScope, mockAgentService, mockWindow, mockNavigationService);
+                mockLocation = { search: function () {
```

Comment:
This works fine, however you can be more consistent with surrounding code by using a jasmine spyObject.